packages/calypso-e2e/src/lib/components/notifications-component.ts

```diff
@@ -59,4 +60,22 @@ export class NotificationsComponent {
 			await this.page.waitForSelector( ':text("Comment trashed")', { state: 'hidden' } );
 		}
```

Comment:
It looks like the `waitForUndoMessage()` function might be fighting with these two lines.

Personally, I think it makes the most sense to just keep the `waitForUndoMessage` function and remove this here - it seems kind of out of place in this function.

But, I do like the selector it is using, so maybe we can re-use that instead of the `.wpnc__summary.wpnc__undo-item` selector.

We'd also need to make sure this doesn't disrupt anywhere that is currently using `clickNotificationAction`